Makefile

```diff
@@ -142,6 +142,18 @@ test-snuba: create-db
 		-vv --cov . --cov-report="xml:.artifacts/snuba.coverage.xml"
 	@echo ""
 
+# snuba-full runs on API changes in Snuba
+test-snuba-full: create-db
+	@echo "--> Running full snuba tests"
+	pytest tests/snuba \
+		tests/sentry/eventstream/kafka \
+		tests/sentry/post_process_forwarder \
+		tests/sentry/snuba \
+		tests/sentry/search/events \
+		tests/sentry/event_manager \
+		-vv --cov . --cov-report="xml:.artifacts/snuba.coverage.xml"
+	pytest tests -m -vv -m snuba_ci
```

Comment:
this command isn't going to work as written -- it's looking for a mark named `-vv`

Reply:
oops, thanks for catching this. fixed in https://github.com/getsentry/sentry/pull/56352